extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfig.java

```diff
@@ -23,6 +26,14 @@ public interface HibernateOrmRuntimeConfig {
     @ConfigDocMapKey("persistence-unit-name")
     Map<String, HibernateOrmRuntimeConfigPersistenceUnit> persistenceUnits();
 
+    /**
+     * Configuration for enabling request scoped sessions
```

Comment:
```suggestion
     * Enable or disable access to a Hibernate ORM `EntityManager`/`Session`/`StatelessSession`
     * *when no transaction is active* but a request scope is.
     *
     * When enabled, the corresponding sessions will be read-only.
     *
     * Defaults to enabled for backwards compatibility, but disabling this is recommended,
     * to avoid inconsistent resulsts caused by queries running outside of transactions.
     *
     * @asciidoctlet
```